pkg/cmd/project/branch/create/create.go

```diff
@@ -0,0 +1,170 @@
+package create
+
+import (
+	"fmt"
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/cmd"
+	sharedBranches "github.com/OctopusDeploy/cli/pkg/cmd/project/branch/shared"
+	"github.com/OctopusDeploy/cli/pkg/cmd/tenant/shared"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/question"
+	"github.com/OctopusDeploy/cli/pkg/util"
+	"github.com/OctopusDeploy/cli/pkg/util/flag"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/projects"
+	"github.com/spf13/cobra"
+)
+
+const (
+	FlagProject    = "project"
+	FlagName       = "name"
+	FlagBaseBranch = "base-branch"
+)
+
+type CreateFlags struct {
+	Project    *flag.Flag[string]
+	Name       *flag.Flag[string]
+	BaseBranch *flag.Flag[string]
+}
+
+type CreateOptions struct {
+	*CreateFlags
+	*cmd.Dependencies
+	shared.GetProjectCallback
+	shared.GetAllProjectsCallback
+	*sharedBranches.ProjectBranchCallbacks
+}
+
+func NewCreateFlags() *CreateFlags {
+	return &CreateFlags{
+		Project:    flag.New[string](FlagProject, false),
+		Name:       flag.New[string](FlagName, false),
+		BaseBranch: flag.New[string](FlagBaseBranch, false),
+	}
+}
+
+func NewCreateOptions(flags *CreateFlags, dependencies *cmd.Dependencies) *CreateOptions {
+	return &CreateOptions{
+		CreateFlags:  flags,
+		Dependencies: dependencies,
+		GetProjectCallback: func(identifier string) (*projects.Project, error) {
+			return shared.GetProject(dependencies.Client, identifier)
+		},
+		GetAllProjectsCallback: func() ([]*projects.Project, error) { return shared.GetAllProjects(dependencies.Client) },
+		ProjectBranchCallbacks: sharedBranches.NewProjectBranchCallbacks(dependencies),
+	}
+}
+
+func NewCreateCmd(f factory.Factory) *cobra.Command {
```

Comment:
Should this be `NewCmdCreate`? Looks like this is the convention we've used elsewhere.